memory/memory/client.py

```diff
@@ -0,0 +1,79 @@
+from asyncinit import asyncinit
+import aiohttp
+import concurrent
+
+from hailtop.auth import service_auth_headers
+from hailtop.config import get_deploy_config
+from hailtop.google_storage import GCS
+from hailtop.tls import get_context_specific_ssl_client_session
+from hailtop.utils import async_to_blocking, request_retry_transient_errors
+
+@asyncinit
+class AsyncMemoryClient:
```

Comment:
Just so we are clear, I imagine this will go away and be replaced with a unified file system interface that has various caching options, yes?

Reply:
Yeah! This was a simple client I wrote, mostly for testing reasons.